### PR TITLE
Communities search: add data to the search results

### PR DESCRIPTION
**Pull request by Pineirin**
closes: zenodo/zenodo-rdm#48

Communities computer tablet search:
Screenshot from 2022-11-10 17-04-58

Communities mobile search:
Screenshot from 2022-11-10 17-05-31

Communities dashboard computer tablet:
Screenshot from 2022-11-10 17-05-48

Communities dashboard mobile:
Screenshot from 2022-11-10 17-06-05

### CODE REVIEW

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js [hunk 1]
```diff
@@ -8,10 +8,11 @@ import { i18next } from "@translations/invenio_app_rdm/i18next";
 import _get from "lodash/get";
 import React from "react";
 import { Image } from "react-invenio-forms";
-import { Icon, Item, Label } from "semantic-ui-react";
+import { Icon, Item, Label, Grid } from "semantic-ui-react";
+import { DateTime } from "luxon";
 
 export const ComputerTabletCommunitiesItem = ({ result }) => {
-  const community_type = result.ui?.type?.title_l10n;
+  const community_type = result.metadata.type?.title[i18next.language];
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js:15]: this is incorrect usage of the translation - the type is already resolved on the backend under .ui as it was implemented before

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js [hunk 2]
```diff
@@ -59,6 +60,10 @@ export const MobileCommunitiesItem = ({ result }) => {
             </a>
           )}
         </Item>
+        <Item.Extra>
+          {i18next.t("Created in: ")}{" "}
+          {DateTime.fromISO(result.created).toLocaleString(i18next.language)}
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js:65]: what kind of metadata is this displaying ? could you include a screenshot?

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js [hunk 3]
```diff
@@ -9,9 +9,10 @@ import _get from "lodash/get";
 import React from "react";
 import { Image } from "react-invenio-forms";
 import { Icon, Item, Label } from "semantic-ui-react";
+import { DateTime } from "luxon";
 
 export const MobileCommunitiesItem = ({ result }) => {
-  const community_type = result.ui?.type?.title_l10n;
+  const community_type = result.metadata.type?.title[i18next.language];
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js:15]: same as above

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js [hunk 4]
```diff
@@ -25,43 +26,59 @@ export const ComputerTabletCommunitiesItem = ({ result }) => {
...
+                    rel="noopener noreferrer"
+                  >
+                    {result.metadata.website}
+                  </a>
+                )}
+              </Item>
+            </Grid.Column>
+            <Grid.Column width={3} className="flex column">
+              <Item.Extra className="text-align-right mt-auto">
+                {i18next.t("Created in: ")}{" "}
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js:74]:
```suggestion
                {i18next.t("Created: ")}{" "}
```

**Review by jrcastro2 — APPROVED** [review]

File: invenio_communities/searchapp.py [hunk 5]
```diff
@@ -21,7 +21,7 @@ def search_app_context():
             config_name="COMMUNITIES_SEARCH",
             available_facets=current_app.config["COMMUNITIES_FACETS"],
             sort_options=current_app.config["COMMUNITIES_SORT_OPTIONS"],
-            headers={"Accept": "application/json"},
+            headers={"Accept": "application/vnd.inveniordm.v1+json"},
```
Comment (kpsherva) [inline invenio_communities/searchapp.py:24]: 👍

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js (hunk 2 above)
Comment (Pineirin) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js:65]: It is now tested. I thought it was bugged but what I thought as a bug was indeed a feature.

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js [hunk 6]
```diff
@@ -8,7 +8,8 @@ import { i18next } from "@translations/invenio_app_rdm/i18next";
 import _get from "lodash/get";
 import React from "react";
 import { Image } from "react-invenio-forms";
-import { Icon, Item, Label } from "semantic-ui-react";
+import { Icon, Item, Label, Grid } from "semantic-ui-react";
```
Comment (zzacharo) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js:11]: The [line](https://github.com/inveniosoftware/invenio-communities/pull/835/files#diff-92f826f7079183b2697c7bbd983065844007d8d44ae983c59ee1bbe910ecd2e5R7) above is wrong! Can you change this?

**Review by kpsherva — APPROVED** [review]

**Pull request merged**